ddsreplayer/test/blackbox/mcap/McapFileReadTest.cpp

```diff
@@ -64,6 +64,13 @@ void create_subscriber_replayer(
 
             std::cout << "replayer created !!!!" << std::endl;
 
+            // Give time for replayer and subscriber match.
+            // Waiting for the subscriber to match the replayer
+            // does not ensure that the replayer is matched the
+            // subscriber so can start replaying messages that
+            // wont arrive to the subscriber
```

Comment:
```suggestion
            // Give time for replayer and subscriber to match.
            // Waiting for the subscriber to match the replayer
            // before starting to replay messages does not ensure 
            // that no samples will be lost (even if using reliable QoS).
            // This is because endpoint matching does not occur
            // at the same exact moment in both ends of communication,
            // so the replayer's writer might have not yet matched the
            // subscriber even if the latter already has (matched the writer).
            // Transient local QoS would be a solution for this,
            // but it is not used as it might pollute frequency arrival
            // measurements.
```